shopinvader_sale_coupon/services/abstract_sale.py

```diff
@@ -15,21 +15,15 @@ def _convert_one_sale(self, sale):
         res = super()._convert_one_sale(sale)
         res.update(
             {
-                "promo_code": sale.promo_code,
+                "promo_codes": sale.promo_codes,
```

Comment:
The promo code disappeared on the `sale.order`. Now several promo codes can be applied at the same time and the codes can be found on the applied coupons and rules (`applied_coupon_ids` and `code_enabled_rule_ids`).
Since several promo codes can be applied, we can now return a string that is a list of the applied promo codes (and an empty string if no promo code is applied). It will minimize the difference in the services.